### PR TITLE
Added elasticsearch hosts and version in configurations files

### PR DESCRIPTION
**Pull request by gram7gram**
#41 fixed

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -68,7 +68,11 @@ swiftmailer:
 
 # MassiveSearch Configuration
 massive_search:
+    adapter: elastic
     adapters:
+        elastic:
+            hosts: '%elasticsearch.hosts%'
```
Comment (alexander-schranz) [inline app/config/config.yml]: demo should work without any problems and as normally a demo is installed on a dev PC without a cluster only  a single host should be in the parameters.yml so we can preconfigure `127.0.0.1:9200` in the paramters.yml.dist file:

```yml
            hosts:
                - "%elasticsearch_host%"
```

File: app/config/config.yml [hunk 2]
```diff
@@ -68,7 +68,11 @@ swiftmailer:
...
+            version: '%elasticsearch.version%'
```
Comment (alexander-schranz) [inline app/config/config.yml]: use `elasticsearch_version` so its the same syntax as for database.

File: app/config/config.yml [hunk 3]
```diff
@@ -177,11 +181,13 @@ ongr_elasticsearch:
     managers:
         default:
             index:
+                hosts: '%elasticsearch.hosts%'
```
Comment (alexander-schranz) [inline app/config/config.yml]: here the same:

```yml
                hosts:
                    - "%elasticsearch_host%"
```

File: app/config/config.yml [hunk 4]
```diff
@@ -177,11 +181,13 @@ ongr_elasticsearch:
...
                 index_name: sulu_rocks_articles
             mappings:
                 - SuluArticleBundle
         live:
             index:
+                hosts: '%elasticsearch.hosts%'
```
Comment (alexander-schranz) [inline app/config/config.yml]: here the same:

```yml
                hosts:
                    - "%elasticsearch_host%"
```

File: app/config/parameters.yml.dist [hunk 5]
```diff
@@ -22,3 +22,8 @@ parameters:
 
     websocket_port: 9876
     websocket_url: 'sulu.lo'
+
+    # Specify elasticsearch configuration
+    # Hosts should be array of strings like: 'http://master.elasticsearch.dev:9200'
+    elasticsearch.hosts: []
```
Comment (alexander-schranz) [inline app/config/parameters.yml.dist]: preconfigure so it works out of the box when elasticsearch is running:

```yml
    elasticsearch_host: 127.0.0.1:9200
```

File: app/config/parameters.yml.dist [hunk 6]
```diff
@@ -22,3 +22,8 @@ parameters:
...
+    elasticsearch.version: 5.6
```
Comment (alexander-schranz) [inline app/config/parameters.yml.dist]: use `_` so its the same syntax as for the database configuration:

```yml
    elasticsearch_version: 5.6
```

**Comment by gram7gram** [comment]
Not sure your were notified, branch it now fits requirements.

**Comment by wachterjohannes** [comment]
@gram7gram thanks for the improvement :)

**Pull request merged**